Group_Constitution

```diff
@@ -0,0 +1,127 @@
+
+The CRO shall select three (3) election dates before March 30th for the voting period. These dates will be announced in a minimum of two (2) weeks prior to elections dates and must fall on weekdays.
+
+The CRO and Scrutinizers shall provide each U of T member with a paper ballot on the voting dates and ask the member to place their ballot in an enclosed box. 
+
+In preparation for a tie, the CRO shall select an executive from amongst the executive committee, to cast their ballot and seal it in an envelope. In the event of a tie for an executive position, one of the two Scrutinizers shall break the seal and count the ballot in order to break the tie.
+
+After the election is over, the CRO and Scrutinizers shall count the ballots. The candidate with the most votes shall be elected to the position. The CRO and Scrutinizers shall submit a report of the results of the elections to the Executive Committee and general members. 
+
+Registered U of T members may not vote by proxy. Non-U of T members may not nominate or vote in elections. 
+
+Only U of T members who have paid any applicable membership fees and have been a member in good standing for 30 days prior to election dates are eligible for voting.
+
+Term of executive positions shall be from May 1st to April 30th.
+
+6.	Finances
+
+The Treasurer shall keep records of all income and expenses. The Treasurer shall present the group’s financial health at the annual general meetings. The Executive Committee will vote on expenditures of over $100.00 by majority vote at an executive meeting. 
+
+The group’s executive or members may not engage in activities that are essentially commercial in nature. This is not intended to preclude the collection of membership fees to cover the expenses of the group, or of charges for specific activities, programs or events, or to prohibit groups from engaging in legitimate fundraising. However, the group will not have as a major activity a function that makes it an on-campus part of a commercial organization, will not provide services and goods at a profit when that profit is used for purposes other than those of the organization, and will not pay salaries to some or all of its officers.
+
+7.	Meetings
+
+A) Annual General Meetings (AGMs):
+
+The group shall hold general meetings at least twice per year, i.e. once per academic term.
+
+The Executive Committee will announce these dates two (2) weeks prior to holding the meetings.  These meetings are intended to go over the group’s annual activity plan, financial health and propose or vote on constitutional amendments, if any. Motions will require 2/3 majority of registered members in attendance for a vote to be cast. The motion with the most votes will be passed. 
+
+b) Executive Meetings:
+
+The executive committee shall meet on a monthly basis where date and times are to be set by an executive.  The quorum of executive meetings shall be 50%+1 of executives.
+
```

Comment:
I don't think we necessarily need in-person executive meetings that are above and beyond the normal group meetings.